conf/settings.py

```diff
@@ -449,3 +449,6 @@
 DIRECTORY_FORMS_API_SENDER_ID_EUEXIT = env.str(
     'DIRECTORY_FORMS_API_SENDER_ID_EUEXIT'
 )
+EUEXIT_AGENT_EMAIL = env.str('EUEXIT_AGENT_EMAIL')
+EUEXIT_GOV_NOTIFY_TEMPLATE_ID = env.str('EUEXIT_GOV_NOTIFY_TEMPLATE_ID')
+EUEXIT_GOV_NOTIFY_REPLY_TO_ID = env.str('EUEXIT_GOV_NOTIFY_REPLY_TO_ID', None)
```

Comment:
Should this default to `None`?

Reply:
si. this results in the default "reply to" email to be used